assets/js/dist/blog-feed.min.js

```diff
@@ -0,0 +1,77 @@
+!(function(e) {
```

Comment:
You seem to be committing some `.min.js` files for the first time. These should be ignored. Did you forget to change a line in `.gitignore`?

Reply:
Ah, it'll be this line: https://github.com/anthonydillon/jaas.ai/blob/move-static-to-assets/.gitignore#L62